main.go

```diff
@@ -34,3 +36,16 @@ func main() {
 		fmt.Println("http.http.ListenAndServe Error", err)
 	}
 }
+
+func getHTTPDownloadURL(p github.PushPayload) string {
+	return "git::" + p.Repository.URL
+}
+
+func downloadRepo(downloadURL string) {
+	err := getter.Get("./repo", downloadURL)
```

Comment:
Maybe give this function a parameter indicating where to put the download

Reply:
this makes sense, so I added it